### ISSUE TITLE
'About Cookies' modal accessibility

### ISSUE THREAD

**Issue by dandalpiaz** [app-illinois/Design-Resources#192]
Below are a few findings when testing the [cookie banner](https://app-illinois.github.io/Design-Resources/Cookie-Banner.html) with a screen reader and keyboard:

- [x] When the 'About Cookies' button is clicked/activated and the modal is opened, nothing is announced by the screen reader. I think a typical pattern is to manually move focus to the first heading in the modal after it is activated, like the second example on this [Deque University: Dialog](https://dequeuniversity.com/library/aria/simple-dialog) page.
- [x] When using the keyboard to navigate backwards through the modal (Shift + Tab), the focus gets 'stuck' on the 'X' button at the top. I think the expected behavior would be to loop back to the last focusable item in the modal; in this case, the 'Close' button. 
- [x] In the modal, there are the two 'close' buttons. The 'X' button at the top has an accessible label of 'Close cookie notice' and the 'Close' button at the bottom has a label of 'Close slideover'. In both cases, it looks like those buttons will close the slideover modal, but do not dismiss the cookie banner on the page. Since the button in the cookie notice itself that is also labeled 'Close Cookie Notice' will dismiss the notice, I wondered if that 'X' button in the modal should do the same?
- [x] In the modal, when the 'University of Illinois System Cookie Policy' link has focus or is hovered, the orange text color of the link does not provide sufficient contrast with its background. Looks like the current ratio is 3.04:1 and at least 4.5:1 would be required for that text size. 

For reference, I used the NVDA screen reader on Windows and the Chrome browser.

**Comment by edthedev** [comment]
Thanks @dandalpiaz! I believe we can address these.

**Comment by edthedev** [comment]
Backwards navigation fixed by #182. Thank you @fleming7 !

**Comment by edthedev** [comment]
@dandalpiaz I installed NVDA, but I was not able to confirm our fix for reading the title - but I also can't get NVDA to read much in-browser to me. Please let us know if the issue persists and we can revisit.

**Comment by dandalpiaz** [comment]
Thanks @edthedev , before, I was testing the https://app-illinois.github.io/Design-Resources/Cookie-Banner.html page, but that page is still showing the same issues. Looks like the more recent deploys may have failed for the GitHub Page site, https://github.com/app-illinois/Design-Resources/deployments ?

**Comment by edthedev** [comment]
@dandalpiaz Oh. Thanks! We will get the deployment fixed.

**Comment by edthedev** [comment]
@dandalpiaz We've fixed the deployment and the updates are available at. https://app-illinois.github.io/Design-Resources/Cookie-Banner.html
